pangram/pangram.sml

```diff
@@ -0,0 +1,2 @@
+fun isPangram (sentence: string): bool =
```

Comment:
Same thing here as in the difference of squares pull request—we don't have these failing stubs elsewhere, and I would favor consistency unless you get a discussion going and there's a consensus that this would be desirable in the SML track.